### DISCUSSION TITLE
Suggestion: Update of SIC Code via Import

### DISCUSSION THREAD

**Discussion by stevearbour** [ledgersmb/LedgerSMB#7091] [Ideas]
I use version 1.7.7, 

I would like to be able (with a checkbox) to update existing SIC Code in the database via importation of a CSV file. So if a country update it's codes, we can generate a new CSV file, and then any existing code in the database should be overwritten instead of being deleted, technically allowing for custom entry to remain if any were manually created after an import ?

I hope this is clear enough  (ツ)

**Comment by ehuelsmann** [comment]
Related to #6606
